brainstormforce/astra
# AST-3610 - Page Background not reflecting correctly in Editor.

### Description

- The Page Background in editor is not reflecting correctly in editor for Content Bg in some cases.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3610
### Screenshots

https://d.pr/v/TsAUIZ

### Types of changes




Bug Fix

### How has this been tested?

- On fresh setup Import [these](https://d.pr/f/ivJjxj) settings.
- Install and activate Spectra to reproduce.
- Set background color site background color and content background color.
- You’ll notice in the editor the color is not inheriting.

### Checklist:
